requirements/customer/glossary.tex

```diff
@@ -53,7 +53,7 @@
         \item One of:
         \begin{itemize}
           \item A Dockerfile providing instructions to build program environment.
-          \item The name of predefined container environment (TODO: link to list of environments we want to provide, in the WMC)
+          \item The name of predefined container environment, for ex. a base image from Docker Hub.
```

Comment:
"for ex." is not a common abbreviation for "for example" I think.
I would suggest:

for ex. -> e. g.
or
for ex. -> for example